package.json

```diff
@@ -21,14 +21,13 @@
     "fmt": "solhint --max-warnings 0 \"contracts/**/*.sol\" && solhint \"test/**/*.sol\" && eslint ./scripts/*.js && eslint \"test/**/*.js\" --no-error-on-unmatched-pattern",
     "migrate": "npm run clean && npx truffle migrate",
     "migrate:flattened": "npm run clean:flattened && npm run flatten && npx truffle migrate --config truffle-config-flattened.js",
-    "postinstall": "node scripts/postinstall.js",
     "test": "npx truffle test",
     "toolkit": "npx witnet-toolkit",
     "try": "npm run compile:requests && npx witnet-toolkit try-data-request --from-solidity ./contracts/requests/"
   },
   "dependencies": {
     "witnet-ethereum-bridge": "~0.4.2",
```

Comment:
I think we should wait to have all the PRs in the bridge merged to know exactly the latest version

Reply:
I think package versions of different repos don't need to match, so I would just approve this PR and let go